lib/deserializers.php

```diff
@@ -182,13 +182,19 @@ function valueObject(Reader $reader, $className, $namespace) {
         return $valueObject;
     }
 
+    $defaultProperties = get_class_vars($className);
+
     $reader->read();
     do {
 
         if ($reader->nodeType === Reader::ELEMENT && $reader->namespaceURI == $namespace) {
 
             if (property_exists($valueObject, $reader->localName)) {
-                $valueObject->{$reader->localName} = $reader->parseCurrentElement()['value'];
+                if (is_array($defaultProperties[$reader->localName])) {
```

Comment:
We could also check if `valueObject->{reader->localName}` is an array and append to instead. Then we wouldnt need to retrieve the class-vars beforehand, wdyt?